README.md

```diff
@@ -124,6 +126,28 @@ If you want to setup the toolchain manually, you can set `rust-toolchain` to `ma
     rust-toolchain: manual
 ```
 
+### Build for target other than default
+
+By default, the action uses the default target (based on the host platform). You can force using a different target (which will be installed if missing) using `rust-target`:
+```yaml
+- name: Check semver
+  uses: obi1kenobi/cargo-semver-checks-action@v2
+  with:
+    rust-target: aarch64-apple-darwin
+```
+If you want to setup the toolchain manually, you can set `rust-toolchain` to `manual`. In this case, the action assumes the specified target is already installed and uses it:
```

Comment:
```suggestion
If you want to set up the toolchain manually, you can set `rust-toolchain` to `manual`. In this case, the action will *not* attempt to install the target — instead, it assumes the target is already set up:
```

Just to emphasize explicitly that manual toolchain selection means the user is on the hook for making sure the target is present too. Otherwise, if there's a 1% chance that a user might misunderstand this, that 1% is a lot of confusion across a large population of users so it'll cause a lot of frustration and "bug report" issues 😅